frontend/src/components/video-chat/video-comment/video-comment.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'common/types/types';
+import { Comment } from 'shared/src/common/types/comment';
+
+import styles from './video-comment.module.scss';
+
+type Props = {
+  comment: Comment;
+};
+
+const VideoComment: FC<Props> = ({ comment }) => {
+  return (
+    <div className={styles['video-comment']}>
+      <div className={styles['main-part-comment']}>
+        <img className={styles['commentators-avatar']} src={comment.avatar} alt={comment.name} />
```

Comment:
Having in mind that user avatar is not required field. So some users do not have avatar. You need to handle that and have default icon when avatar does not exist